### PR TITLE
alert component done

### PR DESCRIPTION
**Pull request by KaustubhM1997**
image
image

### CODE REVIEW

File: alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
+@import url(/CSS/nav.css);
+
+:root {
+
```
Comment (mimansha-swarup) [inline alert/alert.css:4]: declare root variables in one file and then use that no need to declare it in every CSS file

File: alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+.outline-primary,
+.outline-error,
+.outline-success,
+.outline-warning {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0.5rem;
+    box-shadow: 0 0 15px 5px rgb(218, 218, 218);
```
Comment (mimansha-swarup) [inline alert/alert.css:111]: you can also make variable for box-shadow

**Review by mimansha-swarup — COMMENTED** [review]
rest looks good

**Review by manojasarsa — APPROVED** [review]
Looks Good 👍

**Pull request closed without merging**